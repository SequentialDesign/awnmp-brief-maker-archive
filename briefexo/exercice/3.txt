Now, you want to make some phrasse, after all, you could make all the previous things as easily, if not easier by had.

A classical phrasial example is pronoun + verbs

Make a pre-dictionary on which you can:
- write "i" for "I", "u" for "you", "s" for "she", "h" for "he", "t" for "they"
- On top of that, after there might be "m" for "must", "ho" for "should", "ha" for "shall", "c" for "can", "co" for "could".

Be careful, don't forget the space between the pronouns and the verbs
