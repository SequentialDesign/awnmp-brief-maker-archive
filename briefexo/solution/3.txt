(i: I|y: you|h: he|s: she|t: they)
(| " " (m: must | ho: should | ha: shall | c: can | co: could))
