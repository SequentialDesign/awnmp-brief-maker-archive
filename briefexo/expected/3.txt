::i::I
::y::you
::h::he
::s::she
::t::they
::im::I must
::ym::you must
::hm::he must
::sm::she must
::tm::they must
::iho::I should
::yho::you should
::hho::he should
::sho::she should
::tho::they should
::iha::I shall
::yha::you shall
::hha::he shall
::sha::she shall
::tha::they shall
::ic::I can
::yc::you can
::hc::he can
::sc::she can
::tc::they can
::ico::I could
::yco::you could
::hco::he could
::sco::she could
::tco::they could
